### ISSUE TITLE
Icon Request: fa-cthulhu

### ISSUE THREAD

**Issue by StoneCypher** [FortAwesome/Font-Awesome#9351]
Because ia, ia, Cthulhu ftagn phnglui mglwnfah cthulhu rlyeh wgahnagl fhtagn

**Comment by PhillUpNorth** [comment]
As it says in the Necronomicon, 'That which is dead can eternal lie, but with strange aeons even Cthulhu needs an icon', +1 from me!

**Comment by paul-HPA** [comment]
+1!!

**Comment by smalljon** [comment]
Indeed, we must have this icon unless we're willing to disrespect our master.

**Comment by zamber** [comment]
Cthulhu all the things!

**Comment by JoshuaDoshua** [comment]
**What can it be used for?**
games, books, scifi, "lovecraftian" categories. a huge cult favorite

**Example image(s) (optional)**
<img src="https://i.pinimg.com/originals/6c/92/36/6c923664f3c5fa0715fa82627a2fa04e.png" />

<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT_tp-lyUN-dHWsl2SnR2n2UA7uzpgth_nkONW_XeRY5Fkeky6r&s" />

<img src="https://i.pinimg.com/originals/93/c5/35/93c535473a3d22e9f3802e0d3a1aabb5.png" />

<img src="https://www.pngkey.com/png/detail/92-928737_png-cthulhu-svg.png"/>

<img src="https://icon2.cleanpng.com/20180410/kdq/kisspng-the-call-of-cthulhu-t-shirt-decal-sticker-logo-cthulhu-5accb53c6178f9.5007849915233651803993.jpg" />

**Icon request checklist**
- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by somethingSTRANGE** [comment]
This would be useful on blog posts as a warning. "Read at your own risk! May cause insanity." +1

**Comment by StoneCypher** [comment]
I honestly forgot about this issue.

S͕͓͚̦̗͖t̰̞͉̯i͚̮͔͚̺ll̪̮̯̱̣̱̦, I think it would be a good idea, given that the other R̅̌͐ͮẺLI̍̈͆̃ͥ̚Gͮ̽̑IÒͩ̇͋͗Nͣ̇S̉̉ͯ have been shown respect, to pay heed

for lo, unicode will never make me crack.  the `<center>` cannot hold

tony is at T̥̟̖̭͎̟̻͕̱̺̹̠͉̜̺̑̄ͯ̃͂͆͐ͤ͋͒ͅH̲͔̞͑̄̓̀͑ͤ̅̽͂̑́ͨ̾ͨE̻̺̞̖̰̠̖̼͓͗ͤ́̂̿̔ ͈̙̙̗̯̰̘̟͈̞̩̻̱̠͚̰̜̰̣͋͋ͩͣͯ͑ͩͣ͒̆̓̃͌Ḋ̦̮͕̭̣̩̠̹͇̟̜̯̱̾̄ͤ̀͆ͬ͛̃͊ͬ̈̄̓̓̈́̅ͫO̗͕̗̙̣̬̤̹͙̟͚̬̺̦ͬ͛ͭ̆͆ͮͤͩ͌͑̈́O̭̜͈̹͈̠̼͈̘̙͂͗͐́ͪͤͩ̃̚R̹̯̼̪̩̜ͧͦ̓̆̀͐̆̓ͭͬ͒

open the door

open it

the door

**Comment by benjamin-Keller** [comment]
Any updates on this? We'd love to use it as an easter egg on our site!

**Comment by StoneCypher** [comment]
@tagliala - as of today, this is a pro request.

**Comment by xtroce** [comment]
how to upvote this? 
everybody needs cthulhu as an icon, otherwise the world will surely suffer. Also while we are at it, maybe hastur as well? or nyarlathotep ...

**Comment by StoneCypher** [comment]
> how to upvote this? 

FontAwesome is [based on a voting process](https://fontawesome.com/leaderboard/new).  To vote for this, hit 👍 (or any other react) on the base post.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by StoneCypher** [comment]
ị̬̰͇̣̖̗t ͓̗͇͓͕̳̜̦̹is͉͎͚̩̹̭͎̰͕̩͍̜̥ g̭̺̦̲̺̯̪̼̙̳̯ͅoo̘͉̻̭̘̯̘̙̙͈̤̗̺̦d̲̯̰̙̞̜̘̺̖͇̮͓̬̠.̙̣̟̙̻͚̺̮͓  ͉͈̞̺̼̹͕t̳͕̦̤̼͚̞̩̩̪̱ͅo͙̘̞͙͚̱̳͉͙ͅny ̣̲͓͓̙̱the ͔̙̝͉̦̹p̣̘̥͍̤̘̗̙͚̤̯̘͔ͅo̲ṋ̯͍̠̥̣̱y͕̭̠̟͇̱̟ͅ ͚ma̭̭̮̱͇̻̘̬͓̖̥̻̮ͅy͍͇̦̲͚̯̠̻͖ y̩̪͖̫͕͇e̙̥̤̹͈̼̬̰̱͖͇̹̼̭͍t̪̹͇̜̳̲̙͎ ̳̫̬kn̖̼͔̞͇̭͔̦̫̯͍͓ͅo̜͙̜̙̬ͅͅc̬̮̝̟k̙̘̯̳ a̞̙̟̣̘͇̩̯̠̳̰̝̦ț ̹͙̣̥͓̞̩̫͎͖a ͙̝̣̟͇͉̭̞͙͚̜̮͖d̯̥̣if͓̳̬͖͓͎̥̥̻̹͎̥͍͚f͈͉̝̻̫̘̰̻̘̘̦e̹r͖̝͉̫en̥͕̳͈̭͓̫̜̹͉̖ṭ̠͇̰̠̬ ̜̟̟̜͈͉̙̙͕̭̱̱d̻͉̩̠̰̙̟̰͕͈͙̙̙̜̱o̦̟͖o̺r̭̙̙̠͓̪̤̜̩̣͕.

**Comment by StoneCypher** [comment]
@sensibleworld - dead cthulhu may lie sleeping, but I do not

It has been ten years since this paying user requested this icon, and got more support than many icons that have been implemented

Please fix
